pkg/indexmanagement/reconcile.go

```diff
@@ -67,10 +66,10 @@ func RemoveCronJobsForMappings(apiclient client.Client, cluster *apis.Elasticsea
 	for _, mapping := range mappings {
 		policy := policies[mapping.PolicyRef]
 		if policy.Phases.Hot != nil {
-			expected.Insert(fmt.Sprintf("%s-rollover-%s", indexManagmentNamePrefix, mapping.Name))
+			expected.Insert(fmt.Sprintf("%s-roll-%s", cluster.Name, mapping.Name))
```

Comment:
i think `rollover` is still good to keep here, it makes more sense to read than `roll`